packages/core/template_project/app/src/main/AndroidManifest.xml

```diff
@@ -61,6 +76,20 @@
                 android:value="<%= metadata.value %>" />
         <% } %>
 
+        <% if (isMetaQuest) { %>
```

Comment:
@andreban

This is the third `if (isMetaQuest)` in this file. Should I refactor this to a feature like https://github.com/GoogleChromeLabs/bubblewrap/blob/main/packages/core/src/lib/features/ArCoreFeature.ts?

Why `arcore` option is a "feature", but `chromeosonly` option is just an "if condition" (https://github.com/GoogleChromeLabs/bubblewrap/blob/main/packages/core/template_project/app/src/main/AndroidManifest.xml#L29), while both add only one string to `AndroidManifest.xml`?

When should an "if condition" be used and when should a "feature" be used?

Reply:
I think it's down to timelines - Chrome OS only was added before the Feature system was a thing. We should probably migrate.